systemtest/src/test/java/io/strimzi/systemtest/KafkaST.java

```diff
@@ -1169,7 +1171,7 @@ void testKafkaJBODDeleteClaimsTrueFalse() {
         // kafka cluster already deployed
         verifyVolumeNamesAndLabels(2, 2, 10);
         LOGGER.info("Deleting cluster");
-        cmdKubeClient().deleteByName("kafka", CLUSTER_NAME).waitForResourceDeletion("pod", KafkaResources.kafkaPodName(CLUSTER_NAME, 0));
+        cmdKubeClient().deleteByName("kafka", CLUSTER_NAME).waitForResourceDeletion("Kafka", CLUSTER_NAME);
```

Comment:
Are you sure this is sufficient ? I think we can hit there race-condition where pods will be in terminating, same for PVC but Kafka CR will be deleted.